### ISSUE TITLE
Snapdragon S8 SamPWND boot loop logcat - v88 logcat

### ISSUE THREAD

**Issue by AngelIsL33T** [rovo89/XposedBridge#195]
I know samsung stock roms have known issues, but i just thought that i should post this anyway to see if it helps.

https://pastebin.com/bWcXKL5E

**Comment by rovo89** [comment]
```
10-09 08:45:49.870  8875  8875 I Xposed  : Starting Xposed version 88, compiled for SDK 24
10-09 08:45:49.870  8875  8875 I Xposed  : Device: SM-G950U (samsung), Android version 7.0 (SDK 24)
10-09 08:45:49.870  8875  8875 I Xposed  : ROM: NRD90M.G950USQS1AQI6
10-09 08:45:49.870  8875  8875 I Xposed  : Build fingerprint: samsung/dreamqltesq/dreamqltesq:7.0/NRD90M/G950USQS1AQI6:user/release-keys
10-09 08:45:49.870  8875  8875 I Xposed  : Platform: arm64-v8a, 64-bit binary, system server: yes
10-09 08:45:49.870  8875  8875 I Xposed  : SELinux enabled: yes, enforcing: no
```

```
10-09 08:45:46.919  8308  8308 E System  : Unable to open boot classpath entry: /system/framework/oem-services.jar
10-09 08:45:46.920  8308  8308 E System  : java.util.zip.ZipException: File too short to be a zip file: 0
10-09 08:45:46.920  8308  8308 E System  : 	at java.util.zip.ZipFile.<init>(ZipFile.java:208)
10-09 08:45:46.920  8308  8308 E System  : 	at java.util.zip.ZipFile.<init>(ZipFile.java:147)
10-09 08:45:46.920  8308  8308 E System  : 	at java.util.jar.JarFile.<init>(JarFile.java:161)
10-09 08:45:46.920  8308  8308 E System  : 	at java.util.jar.JarFile.<init>(JarFile.java:98)
10-09 08:45:46.920  8308  8308 E System  : 	at libcore.io.ClassPathURLStreamHandler.<init>(ClassPathURLStreamHandler.java:47)
10-09 08:45:46.920  8308  8308 E System  : 	at java.lang.VMClassLoader.createBootClassPathUrlHandlers(VMClassLoader.java:49)
10-09 08:45:46.920  8308  8308 E System  : 	at java.lang.VMClassLoader.<clinit>(VMClassLoader.java:31)
10-09 08:45:46.920  8308  8308 E System  : 	at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:742)
10-09 08:45:46.920  8308  8308 E System  : 	at java.lang.BootClassLoader.loadClass(ClassLoader.java:1403)
10-09 08:45:46.920  8308  8308 E System  : 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
10-09 08:45:46.920  8308  8308 E System  : 	at java.lang.Class.classForName(Native Method)
10-09 08:45:46.920  8308  8308 E System  : 	at java.lang.Class.forName(Class.java:400)
10-09 08:45:46.920  8308  8308 E System  : 	at external.org.apache.commons.lang3.ClassUtils.getClass(ClassUtils.java:823)
10-09 08:45:46.920  8308  8308 E System  : 	at de.robv.android.xposed.XposedHelpers.findClass(XposedHelpers.java:67)
10-09 08:45:46.920  8308  8308 E System  : 	at de.robv.android.xposed.XposedHelpers.getParameterClasses(XposedHelpers.java:508)
10-09 08:45:46.920  8308  8308 E System  : 	at de.robv.android.xposed.XposedHelpers.findAndHookMethod(XposedHelpers.java:183)
10-09 08:45:46.920  8308  8308 E System  : 	at de.robv.android.xposed.XposedInit.initForZygote(XposedInit.java:95)
10-09 08:45:46.920  8308  8308 E System  : 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:89)
```

@wanam Did you have that, too?

```
10-09 08:45:47.468  8308  8861 E art     : No implementation found for void dalvik.system.VMRuntime.setSystemDaemonThreadPriority() (tried Java_dalvik_system_VMRuntime_setSystemDaemonThreadPriority and Java_dalvik_system_VMRuntime_setSystemDaemonThreadPriority__)
```
Looks like they have backported https://github.com/rovo89/android_art/commit/c7a1314eec05bc1a8300165b24b62a2dc587e882 to SDK24. I have ported it to SDK25 in a compatible way, so I can easily do that for SDK24 as well.

**Comment by AngelIsL33T** [comment]
@rovo89 Thanks for reviewing the file. I'll let you do you from here on! Hopefully it'll be working soon.

**Comment by wanam** [comment]
@rovo89 No, i guess it was backported to some S8 FWs only, because it was reported working by some other s8 sdk24 users.

**Comment by rovo89** [comment]
And the "Unable to open boot classpath entry: /system/framework/oem-services.jar"?

**Comment by wanam** [comment]
@rovo89  No i don't have that error, neither the file "/system/framework/oem-services.jar" on my GS7, @AngelIsL33T  is it a deodexed Rom?

**Comment by AngelIsL33T** [comment]
@wanam no. Standard stock img using SamPWND root

**Comment by Firefds** [comment]
@wanam @robo89, this error is happening on the Snapdragon version of the galaxy s8.
The exynos version waa fixed with your exynos-m2 fix.

**Comment by AngelIsL33T** [comment]
@rovo89 Do you think there's any solution to the problem with the log I posted? Sorry if your busy or whatnot. Don't mean to be a bug

**Comment by Firefds** [comment]
@rovo89 @wanam  - here is a link for the Snap S8 Framework directory
https://www.dropbox.com/s/zw9zy9ej4pc815m/framework.zip?dl=0

**Comment by AngelIsL33T** [comment]
@Firefds This isn't for me to flash on my device right?

**Comment by Firefds** [comment]
No.. it's just a zip of the framework folder

**Comment by AngelIsL33T** [comment]
@Firefds ok lol. Just making sure :p

**Comment by rovo89** [comment]
Should be fixed with v88.1, now available. Please reopen if the issue still exists.
